doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -84,100 +92,62 @@ This is an example configuration that was used to ingest the Gaia DR2 catalog:
     # while the Gaia-specific class handles the errors.
     config.mag_column_list = ["phot_g_mean", "phot_bp_mean", "phot_rp_mean"]
```

Comment:
I don't think this is correct, and this confused me.  These columns are special-cased by the gaia manager, and do not go through the regular code (which would be wrong because the gaia magnitude columns are Vega, while the regular code assumes AB).  Thankfully, I think that the code is correct but this config is confusing because it's not used as best as I can tell.

Reply:
I updated the comment on this required column to make it more clear.